### ISSUE TITLE
Refresh groups doesn't work ?

### ISSUE THREAD

**Issue by Kize** [angular-gantt/angular-gantt#443]
Hi,

When I remove a task from a row, groups doesn't refresh at all.

I tried to force the call to `refresh()` :

```
api.tasks.on.remove($scope, function () {
                api.groups.refresh();
              });
```
But it still doesn't refresh groups position.

I remove the task with a `#splice()`, and I checked, the function `api.tasks.on.remove()` is called.

**Comment by Toilal** [comment]
Did you solved your issue ?

**Comment by Kize** [comment]
No sorry, and i'm not currently working on it.

**Comment by wangxu-alvin** [comment]
Hi, 
I found the similar problem. I want to delete a task by clicking the task.
`var deleteTask = function (task) {`
`    if (!$scope.options.readOnly) {`
`        dataToRemove = [{'id': task.row.model.id, 'tasks': [{'id': task.model.id}]} ];`
`        $scope.api.data.remove(dataToRemove);`
`        dataToRemove = undefined;`
`        $scope.api.groups.refresh();`
`    }`
 `}`
After a task is removed, the corresponding bar in the group row  is not deleted until drawing a  new taks on the gantt chart. However, the group can be refreshed without $scope.api.groups.refresh(); at the end of deleteTask function which I have tried.
